nibabel/cifti2/parse_cifti2.py

```diff
@@ -121,33 +121,9 @@ def _chk_pixdims(hdr, fix=False):
 
 
 class _Cifti2AsNiftiImage(Nifti2Image):
+    """ Load a NIfTI2 image with a Cifti2 header """
     header_class = _Cifti2AsNiftiHeader
-    files_types = (('image', '.nii'),)
-    valid_exts = ('.nii',)
     makeable = False
-    rw = True
-
-    def __init__(self, dataobj, affine, header=None,
```

Comment:
I guess this is because the `cifti_img` attribute is not being used elsewhere?

Do we want to raise an error if no cifti extension found?

What is this subclass now for?